http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreterSpec.scala

```diff
@@ -201,15 +225,60 @@ class DataprocInterpreterSpec
                                                      true,
                                                      false
     )
+    val resourceConstraints = dataprocInterpHighMem()
+      .getDataprocRuntimeResourceContraints(testClusterClusterProjectAndName,
+                                            runtimeConfig.machineType,
+                                            RegionName("us-central1")
+      )
+      .unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
+
+    val dataProcSoftwareConfig = dataprocInterp().getSoftwareConfig(
+      GoogleProject("MyGoogleProject"),
+      RuntimeName("MyRuntimeName"),
+      runtimeConfig,
+      resourceConstraints
+    )
+
+    val propertyMap = dataProcSoftwareConfig.getPropertiesMap()
+    resourceConstraints.totalMachineMemory shouldBe MemorySize.fromGb(104)
+    resourceConstraints.memoryLimit shouldBe MemorySize((MemorySize.fromGb(104).bytes * (1 - 0.8)).toLong)
```

Comment:
can the .8 here be sourced from the same constraints the interp/`getSoftwareConfig` is given?